libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnector.java

```diff
@@ -133,6 +146,20 @@ public InputStream unsafeGetAttachmentStream(AttachmentContent attachment) throw
         return in;
     }
 
+    private String printAcceptedFileName(String name, int count) {
+        String acceptedFileName;
+        int extensionDot;
+
+        extensionDot = name.lastIndexOf('.');
+        if (extensionDot >= 0) {
+            acceptedFileName = name.substring(0, extensionDot) + String.format("(%d)", count) + name.substring(extensionDot);
```

Comment:
minor comment: This does not add the `" "` before the opening `(` as suggested in the PR comment

Reply:
added a space